FRINXio/frinx-frontend
# update yarn to latest

This PR updates yarn to the latest version (3.2.0).
To test (and later use) everything run:
```
corepack enable 
yarn set version stable
```
See more: https://yarnpkg.com/getting-started/install
